x-pack/plugins/reporting/server/routes/common/get_user.ts

```diff
@@ -5,9 +5,8 @@
  * 2.0.
  */
 
-import { KibanaRequest } from '@kbn/core/server';
-import { SecurityPluginStart } from '@kbn/security-plugin/server';
+import { KibanaRequest, SecurityServiceStart } from '@kbn/core/server';
 
-export function getUser(request: KibanaRequest, security?: SecurityPluginStart) {
-  return security?.authc.getCurrentUser(request) ?? false;
+export function getUser(request: KibanaRequest, securityService?: SecurityServiceStart) {
+  return securityService?.authc.getCurrentUser(request) ?? false;
```

Comment:
The `ReportingInternalStart` interface should have the security service as non-optional, meaning it can be specified as non-optional here:
```suggestion
export function getUser(request: KibanaRequest, securityService?: SecurityServiceStart) {
  return securityService?.authc.getCurrentUser(request) ?? false;
```